libs/components/src/search/vl-search.uig-css.ts

```diff
@@ -128,7 +128,7 @@ const styles: CSSResult = css`
     }
     .vl-search--inline .vl-search__input:focus,
     .vl-search--inline slot[name='input']:focus {
-        transition: width 0.2s;
+        // transition: width 0.2s;
```

Comment:
deze lijn gewoon verwijderen, niet in commentaar zetten

Reply:
Done.